core/shared/src/main/scala-3/org/threeten/bp/DayOfWeek.scala

```diff
@@ -227,23 +230,24 @@ enum DayOfWeek(name: String, ordinal: Int)
   /**
    * Gets the range of valid values for the specified field.
    *
-   * The range object expresses the minimum and maximum valid values for a field.
-   * This day-of-week is used to enhance the accuracy of the returned range.
-   * If it is not possible to return the range, because the field is not supported
-   * or for some other reason, an exception is thrown.
+   * The range object expresses the minimum and maximum valid values for a field. This day-of-week
+   * is used to enhance the accuracy of the returned range. If it is not possible to return the
+   * range, because the field is not supported or for some other reason, an exception is thrown.
    *
-   * If the field is {@link ChronoField#DAY_OF_WEEK DAY_OF_WEEK} then the
-   * range of the day-of-week, from 1 to 7, will be returned.
-   * All other {@code ChronoField} instances will throw a {@code DateTimeException}.
+   * If the field is {@link ChronoField#DAY_OF_WEEK DAY_OF_WEEK} then the range of the day-of-week,
```

Comment:
Are these formatting changes?

Reply:
Yes. Formatting was not being checked for Scala 3 until this PR.